src/App.tsx

```diff
@@ -1,8 +1,83 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+import { getTodos } from './api/getTodos';
+import { getUsers } from './api/getUsers';
+
+import { TodoList } from './components/TodoList';
+import { PreparedTodo, UserType } from './utils/interfaces';
+
+const App: FC<{}> = () => {
```

Comment:
в функциональном компоненте если пропсов нет, то пустой объект не нужен
```suggestion
const App: FC = () => {
```